src/deepsparse/loggers/helpers.py

```diff
@@ -91,6 +139,7 @@ def get_function_and_function_name(
 
     :return: A tuple (function, function name)
     """
+    import deepsparse.loggers.metric_functions as built_ins
```

Comment:
Alternatively, postponing the process of registration of the built-ins until this function is called (we want to fetch all the built-ins (CV, NLP, etc) here on import, so our decorators will be evoked.

Reply:
why do we delay this?